diff --git a/src/ipa/rpi/controller/controller.cpp b/src/ipa/rpi/controller/controller.cpp
index df45dcd3..03ef67d0 100644
--- a/src/ipa/rpi/controller/controller.cpp
+++ b/src/ipa/rpi/controller/controller.cpp
@@ -75,7 +75,7 @@ const std::map<std::string, Controller::HardwareConfig> &hardwareConfigMap()
 				* could happen using 4 lanes at 1.5Gbps at 10bpp with
 				* frames wider than ~16,000 pixels.
 				*/
-				.minPixelProcessingTime = 1.0us / 380,
+				.minPixelProcessingTime = 1.0us / 580,
 				.dataBufferStrided = false,
 			}
 		},
